Makefile.pre.in

```diff
@@ -1293,6 +1297,7 @@ PYTHON_HEADERS= \
 		$(srcdir)/Include/internal/pycore_uop_metadata.h \
 		$(srcdir)/Include/internal/pycore_warnings.h \
 		$(srcdir)/Include/internal/pycore_weakref.h \
+		$(srcdir)/Python/condvar.h \
```

Comment:
As I mentioned in my previous review, this should be a dependency of `ceval_gil.o` (and not `ceval.o`, as it currently is). IOW:

- I don't think it is right to include it in `PYTHON_HEADERS`
- I think we should remove `condvar.h` from the `ceval.o` target
- I think we need a separate target for `ceval_gil.o`

Reply:
If we declare ceval_gil.o as a separate target, what line should it go on?

Reply:
It should be near the `ceval.o` target.

Reply:
I already updated the PR. Could you add a label “skip news”?